chart/stash/templates/update-status-function.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: stash.appscode.com/v1beta1
+kind: Function
+metadata:
+  name: update-status
+  labels:
+    {{- include "stash.labels" . | nindent 4 }}
+spec:
+  image: {{ .Values.operator.registry }}/{{ .Values.operator.repository }}:{{ .Values.operator.tag }}
+  args:
+    - update-status
+    - --namespace=${NAMESPACE:=default}
+    - --repository=${REPOSITORY_NAME:=}
+    - --backup-session=${BACKUP_SESSION:=}
+    - --restore-session=${RESTORE_SESSION:=}
+    - --output-dir=${outputDir:=}
+    - --enable-status-subresource=${ENABLE_STATUS_SUBRESOURCE:=false}
```

Comment:
`ENABLE_STATUS_SUBRESOURCE` is not being set to true by default.

Reply:
This variable is resolved by Stash operator.